pysmurf/tune/smurf_tune.py

```diff
@@ -2214,20 +1991,25 @@ def flux_ramp_check(self, band, reset_rate_khz=4,
 
         return d, df, sync
 
-    def tracking_setup(self, band, channel=None, reset_rate_khz=4., write_log=False, 
-        make_plot=False, save_plot=True, show_plot=True, nsamp=2**19,
-        lms_freq_hz=None, meas_lms_freq=False, flux_ramp=True, fraction_full_scale=None,
-        lms_enable1=True, lms_enable2=True, lms_enable3=True, lms_gain=7,
-        return_data=True, new_epics_root=None,
+    def tracking_setup(self, band, channel=None, reset_rate_khz=4., 
+        write_log=False, make_plot=False, save_plot=True, show_plot=True, 
+        nsamp=2**19, lms_freq_hz=None, meas_lms_freq=False, flux_ramp=True, 
+        fraction_full_scale=None, lms_enable1=True, lms_enable2=True, 
+        lms_enable3=True, lms_gain=7, return_data=True, new_epics_root=None,
```

Comment:
@eyyoung24 Would you mind fixing the reset_rate_khz=4 and lms_gain=7 hardcode defaults here?  lmsGain can be None and if not provided can be taken from the pysmurf config file.  We should do the same for reset_rate_khz - but if you don't want to do that one you can just create a pysmurf issue for adding it to the config file.

Reply:
@swh76 Can you point me to where the lms_gain=None check is? I'm not obviously finding it. Want to make sure it's doing the right thing.

Reply:
@eyyoung24 oh sorry, I just meant to implement code for lms_gain in that (and other) functions like this ; 
https://github.com/slaclab/pysmurf/blob/b397b6d7382fff7794a367225ed80b6a319381f6/pysmurf/util/smurf_util.py#L2462
where if none is explicitly provided, it will default to the value in the pysmurf cfg file.